.circleci/config.yml

```diff
@@ -1122,6 +1120,19 @@ jobs:
       - run_build
       - matrix_notify_failure_unless_pr
 
+  t_ubu_2204_soltest:
```

Comment:
Please add some checks verifying that it's actually running the earliest supported versions of our dependencies documented in [Building from Source](https://docs.soliditylang.org/en/develop/installing-solidity.html#building-from-source) (#15136).

The fact that testing min versions is the point should also be reflected in job name.

Reply:
You mean to parse the minimum versions out of the docs? I feel like this is not the right PR for that. And also not very productive if we're about to bump them anyways.

Reply:
No, that would be insane :) I meant to just take the versions listed there and use then in those checks.

I'm mentioning it because the PR already adds a new CI job. We should finally add this job. This is not the first bug that slipped through due to us not having these checks.

Reply:
Yeah let me do that. I don't know if I'll get to it today but probably some time next week.